engines/sci/graphics/palette32.cpp

```diff
@@ -206,17 +209,25 @@ int16 GfxPalette32::matchColor(const byte r, const byte g, const byte b, const i
 	return bestIndex;
 }
 
-void GfxPalette32::updateForFrame() {
+bool GfxPalette32::updateForFrame() {
 	applyAll();
 	_versionUpdated = false;
 	// TODO: Implement remapping
```

Comment:
Remapping was implemented via RE, and it is working mostly OK. Are there any showstoppers for re-enabling the current code?